tests/IBTK/rt0_refine_01_2d.4.output

```diff
@@ -0,0 +1,24 @@
+test results for constant_function
+max norm of u_sc: 4
+max norm of exact - refined: 0
+test results for linear_x
+max norm of u_sc: 1
+max norm of exact - refined: 0
+test results for linear_y
+max norm of u_sc: 1
+max norm of exact - refined: 0
+test results for both_linear
+max norm of u_sc: 3
+max norm of exact - refined: 0
+test results for non_rt
+max norm of u_sc: 1.76562
+max norm of exact - refined: 0.272461
+test results for non_rt
+max norm of u_sc: 1.93848
+max norm of exact - refined: 0.0697632
+test results for non_rt
+max norm of u_sc: 1.93848
+max norm of exact - refined: 0.0175438
+test results for non_rt
+max norm of u_sc: 1.93848
+max norm of exact - refined: 0.00439239
```

Comment:
I am surprised that this is second order, while `rt0_refine_01_2d.2.output` is first order. @boyceg did we expect this?

Reply:
If the answer is not obvious let me know and I will investigate.

Reply:
I think that this is first order. The refinement ratio is 4, and you see the error go down by a factor of 4 when you refine by a factor of 4.

Reply:
oh, duh. This makes sense.